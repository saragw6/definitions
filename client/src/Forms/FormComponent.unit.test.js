import React from 'react';
import renderer from "react-test-renderer";
import Form from './FormComponent';

test('termRequest?', () => {
  const component = renderer.create(<Form title="Request a Term" inputs={[{labelInput:'What term do you want defined?', isRequired:true}]} />);
  const jsonForm = component.toJSON();
  expect(jsonForm).toMatchSnapshot();
});