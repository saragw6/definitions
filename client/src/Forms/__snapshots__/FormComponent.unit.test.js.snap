exports[`test termRequest? 1`] = `
<div
  className="flex-container"
  data-testid="form-container">
  <form
    data-testid="request-form"
    onSubmit={[Function]}>
    <div
      className="form">
      <div
        className="page-title">
        Request a Term
      </div>
      <div />
      <span
        className="error">
        * Required
      </span>
      <div
        className="inputsBox">
        <div
          className="formInput">
          <label>
            What term do you want defined?
          </label>
          Â 
          <span
            className="error">
            *
          </span>
          <div
            className="queerInput"
            data-react-toolbox="input">
            <input
              className=""
              data-testid="request-input"
              defaultValue={undefined}
              disabled={false}
              maxLength={50}
              name={undefined}
              onBlur={undefined}
              onChange={[Function]}
              onKeyPress={undefined}
              required={false}
              role="input"
              style={
                Object {
                  "background": "transparent",
                }
              }
              type="Text"
              value={undefined} />
            <span
              className={undefined} />
            <span
              className={undefined}
              hidden={undefined}>
              Your answer
            </span>
            <span
              className={undefined}>
              0
              /
              50
            </span>
          </div>
        </div>
      </div>
      <div
        className="submitBox">
        <button
          className=" queerButton"
          data-react-toolbox="button"
          disabled={false}
          href={undefined}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchStart={[Function]}
          type="button">
          Submit
        </button>
      </div>
    </div>
  </form>
</div>
`;
